frontend/src/metabase/common/components/DataPicker/components/TablePicker.tsx

```diff
@@ -28,10 +29,10 @@ interface Props {
    */
   databaseId?: DatabaseId;
   value: TablePickerValue | undefined;
-  onChange: (value: NotebookDataPickerValueItem) => void;
+  onItemSelect: (value: DataPickerItem) => void;
```

Comment:
Renamed `onChange` to `onItemSelect` to make it consistent with other *Picker components props (e.g. `QuestionPicker`).